src/File/UploadFromFile.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * @copyright 2015-2017 Contentful GmbH
+ * @license   MIT
+ */
+
+namespace Contentful\File;
+
+class UploadFromFile implements FileInterface
+{
+    /**
+     * @var string
+     */
+    private $fileName;
+
+    /**
+     * @var string
+     */
+    private $contentType;
+
+    /**
+     * @var \JsonSerializable
+     */
+    private $uploadFrom;
+
+    public function __construct($fileName, $contentType, \JsonSerializable $uploadFrom)
```

Comment:
Because I'm not yet sure how to work with this in the CMA, so for the moment I'm only using `JsonSerializable`